wordpress-mobile/WordPress-Aztec-iOS
# Loads HTML into the demo editor.

**What works:**

Loads HTML into the demo editor.

**What doesn't work:**

Unfortunately there's an issue with how we've implemented the formatting that's making it not work properly.  In any case, since there's value in having the parser wired, I'm submitting this PR now and will work in parallel on a fix for the formatting issues.

**How to test:**

Just launch the demo app, go to "Editor Demo" and make sure the HTML text is loaded.  Some styles may be present, but they will mostly not work yet.
